spring-boot-project/spring-boot-autoconfigure/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -1125,6 +1125,12 @@
         "level": "error"
       }
     },
+    {
+      "name": "spring.data.web.pageable.serialization-mode",
+      "type": "org.springframework.data.web.config.EnableSpringDataWebSupport.PageSerializationMode",
+      "description": "Configures how to render spring data Pageable instances.",
+      "defaultValue": "DIRECT"
+    },
```

Comment:
We can remove this when merging thanks to https://github.com/spring-projects/spring-boot/issues/7562.

Reply:
done.